website/docs/d/synthetics_monitor_location.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_synthetics_monitor_location"
+sidebar_current: "docs-newrelic-datasource-synthetics-monitor-location"
+description: |-
+  Grabs a Synthetics monitor location by label.
+---
+
+# Data Source: newrelic\_synthetics\_monitor
+
+Use this data source to get information about a specific Synthetics monitor locations in New Relic that already exist. This can be used to set up a Synthetics alert condition.
+
+## Example Usage
+
+```hcl
+data "newrelic_synthetics_monitor" "bar" {
```

Comment:
We'll want to replace the example here that is based on alert conditions with one that represents how this might be used in the wild, where the data source can be used to supply a `newrelic_synthetics_monitor` resource with a value for the `locations` attriubute.   Probably something like:

```
data "newrelic_synthetics_monitor_location" "bar" {
  label = "My private location"
}

resource "newrelic_synthetics_monitor" "foo" {
  name = "foo"
  type = "SIMPLE"
  frequency = 5
  status = "ENABLED"
  locations = [data.newrelic_synthetics_monitor_location.bar.name]

  uri                       = "https://example.com"               # Required for type "SIMPLE" and "BROWSER"
  validation_string         = "add example validation check here" # Optional for type "SIMPLE" and "BROWSER"
  verify_ssl                = true                                # Optional for type "SIMPLE" and "BROWSER"
}
```